### ISSUE TITLE
Problem with slim-lint, rubocop and Style/IndentHash

### ISSUE THREAD

**Issue by suchy** [sds/slim-lint#30]
Hi,

I'm using slim-lint and rubocop.

This code:

``` ruby
= select_tag :c,
  data: {\
    a: 'b'}
```

gives me that warning:

```
index.html.slim:3 [W] RuboCop: Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.
```

To be honest I'm not sure what could wrong with my code. Any ideas?

**Comment by sds** [comment]
Hey @suchy, thanks for the report.

There's nothing wrong with your code, it's just that Slim-Lint's method of extracting Ruby code from a Slim template and then scanning that extracted code with RuboCop results in unexpected problems like this.

My recommendation is to disable the `Style/IndentHash` cop for the `RuboCop` linter in your `.slim-lint.yml`.

I'll leave this open for now so that I can have a look at seeing if this is something we can avoid in the general case.

**Comment by D-system** [comment]
Hi @sds 
What's the syntax in `.slim-lint.yml` to disabled Rubocop rules ?
